src/Entity/CitizenProjectSkill.php

```diff
@@ -19,8 +21,16 @@
  */
 class CitizenProjectSkill extends BaseSkill
 {
+    /**
+     * @var Collection|CitizenProjectCategorySkill
+     *
+     * @ORM\OneToMany(targetEntity="AppBundle\Entity\CitizenProjectCategorySkill", mappedBy="skill", orphanRemoval=true)
```

Comment:
Are you making this relation bi-directional just to add the `orphanRemoval` directive?

If so you should create a migration instead, because this is only managed by doctrine, a delete query on the db will have the same problem:
```php
class CitizenProjectCategorySkill
{
    /**
     * ...
     * @ORM\JoinColumn(onDelete="CASCADE")
     */
    private $skill;

// ...
```
Will handle the case using doctrine or the db with the same behavior.